### PR TITLE
CI fixes

### PR DESCRIPTION
**Pull request by pleimer**

### CODE REVIEW

File: plugins/handler/ceilometer-metrics/main.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ func (c *ceilometerMetricHandler) Config(blob []byte) error {
 	return nil
 }
 
-//New ceilometer metric handler constructor
+//  New ceilometer metric handler constructor
```
Comment (paramite) [inline plugins/handler/ceilometer-metrics/main.go:244]: two ws

**Comment by pleimer** [comment]
There's still a couple linter issues that need to be resolved here and I have to figure out the problem with the Linter running in GHA

**Comment by pleimer** [comment]
I have to implement a manual install of the qpid-proton development files in GHA because the ubuntu packages did not install the files where expected. This results in a really long runtime, unfortunately, and most likely a much higher GHA quota usage. Hopefully there is a way to work around this. I will research https://github.com/actions/cache for potential caching

**Comment by pleimer** [comment]
I think this container based approach may actually be the best option: https://www.edwardthomson.com/blog/github_actions_5_building_in_containers.html

**Pull request merged**